starlette/responses.py

```diff
@@ -262,14 +271,36 @@ async def stream_response(self, send: Send) -> None:
 
         await send({"type": "http.response.body", "body": b"", "more_body": False})
 
+        if support_trailers and self.trailers:
+            trailers: typing.List[typing.Tuple[str, str]] = []
+            for key, call in self.trailers.items():
+                if not is_async_callable(call):
+                    trailers.append(key, await anyio.to_thread.run_sync(call))
+                else:
+                    trailers.append(key, await call())
+
+            await send(
```

Comment:
Multiple `http.response.trailers` can be sent, but... Is there a reason for it? Maybe with another kind of API it would make sense... :thinking: